README.md

```diff
@@ -74,16 +74,13 @@ pip3 install -r requirements.txt
 ## Getting started
 Clarifai uses **Personal Access Tokens(PATs)** to validate requests. You can create and manage PATs under your Clarifai account security settings.
 
-[Create PAT:](https://docs.clarifai.com/clarifai-basics/authentication/personal-access-tokens/)
-* ***Log into Portal &rarr; Profile Icon &rarr; Security Settings &rarr; Create Personal Access Token &rarr; Set the scopes &rarr; Confirm***
+* [Create PAT:](https://docs.clarifai.com/clarifai-basics/authentication/personal-access-tokens/) ***Log into Portal &rarr; Profile Icon &rarr; Security Settings &rarr; Create Personal Access Token &rarr; Set the scopes &rarr; Confirm***
+<br />
```

Comment:
![Screenshot 2023-10-30 at 1 16 06 PM](https://github.com/Clarifai/clarifai-python/assets/48971969/5bb7a6b2-3135-4787-b86c-41b1b79a912c)
Maybe remove this <br>? Looks a bit odd to have that much space between 2 bullets.

Reply:
Otherwise, LGTM

Reply:
ok, I will change it.

Reply:
Thanks! Looks good, but it seems like some linting tests are failing. Did you set up your pre-commit hooks?

Reply:
![image](https://github.com/Clarifai/clarifai-python/assets/147609006/5d6d8641-7dbf-4c1d-8bfe-daffb9cb842e)

yes.

Reply:
This looks good to me, thanks for iterating. Just need to figure out why the tests are failing and address @sainivedh 's comments :)